### DISCUSSION TITLE
PHP 7.4 how to deal with dependency problem?

### DISCUSSION THREAD

**Discussion by mq-lukasz** [spatie/browsershot#550] [Q&A] [answered]
After the last update there is a problem with php 7.4 - downloading dependency spatie/image version 2.1.0 which does not support php 7.4. How to deal with it? Downgrading version of browsershot is not a solution.

**Comment by clementmas** [comment, marked as answer]
All the open-source packages Spatie are releasing are written using the latest PHP versions.

So either you upgrade to PHP 8, downgrade your spatie package or use another one.

Note: support for PHP 7.x will end soon: https://www.php.net/supported-versions.php

**Reply by mq-lukasz** [reply]
I updated to php 8.0, thanks.
